pom.xml

```diff
@@ -1993,15 +1993,15 @@
         <mongodb.driver.version>4.1.0</mongodb.driver.version>
 
         <!-- Carbon Identity versions -->
-        <carbon.identity.version>5.24.10</carbon.identity.version>
-        <carbon.identity.governance.version>1.7.2</carbon.identity.governance.version>
-        <carbon.identity-inbound-auth-oauth.version>6.9.8</carbon.identity-inbound-auth-oauth.version>
-        <carbon.identity-oauth2-grant-jwt.version>2.1.1</carbon.identity-oauth2-grant-jwt.version>
-        <carbon.identity-user-ws.version>5.6.1</carbon.identity-user-ws.version>
-        <carbon.identity-data-publisher-application-authentication.version>5.5.1</carbon.identity-data-publisher-application-authentication.version>
+        <carbon.identity.version>5.25.92</carbon.identity.version>
+        <carbon.identity.governance.version>1.8.14</carbon.identity.governance.version>
```

Comment:
1.8.105

Reply:
As mentioned in the PR description this PR only upgrades to the IS 6.1 compatible versions. Component upgrade to the latest versions is addressed in PR https://github.com/wso2/carbon-apimgt/pull/12245/files.